src/Management/Model/SearchEngineList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Doofinder\Management\Model;
+
+use Doofinder\Shared\Interfaces\ModelInterface;
+
+class SearchEngineList implements ModelInterface
```

Comment:
It's not necessary.
This way is ok if we want to add extra functionality to simplify client actions, for example a search function. (I don't know what clients do usually)

Reply:
In that case it may better to not have the class, if we don't have a real usage for this abstraction (at least for now, if we find a usage we can do it)